README.md
Official Website for Manorama School of fine arts

changelog.txt
==CHANGELOG==
1. About Section Width increased
2. Removed Jquery 2.2.0 (redundant) -> Spoiled slick -> Found Fix (https://github.com/kenwheeler/slick/issues/2741)
3. Removed Redundant Bootstrap cdn
4. Fixed Media Section spacing
5. Fixed Slick Slider issue
6. Improved Navbar Logo


js/custom.js
// Document read
(function ($) {
    "use strict";

    var $window = $(window);

//////// -Preloader- /////////
    $window.on('load', function () {
        var loading = $('.loading');
        $(loading).hide(); // Hide preloader when document is fully loaded
    });

//////// -Mobile Menu- /////////
    var icon = $(".icon");
    var mobnav = $(".mobilenav");
    var topmen = $(".top-menu");
    var midmen = $(".mid-menu");
    var botmen = $(".bottom-menu");
    var close = $(".close-menu");
    $(icon).on('click', function () {
        $(mobnav).fadeToggle(500);
        $(topmen).toggleClass("top-animate");
        $(midmen).toggleClass("mid-animate");
        $(botmen).toggleClass("bottom-animate");
    });

    $(close).on('click', function () {
        $(mobnav).hide();
        $(topmen).removeClass("top-animate");
        $(midmen).removeClass("mid-animate");
        $(botmen).removeClass("bottom-animate");

    });


//////// -Smooth Scroll- /////////
    smoothScroll.init({
        selector: '[data-scroll]',
        selectorHeader: '[data-scroll-header]',
        speed: 1000, // Scrolling Time in Milliseconds(ms) // 1000ms = 1sec //
        easing: 'easeInOutCubic',
        offset: 0,
        updateURL: true,
        callback: function (anchor, toggle) {
        }
    });

//////// -Scroll to top button- /////////
    var scrolls = $('.scrollToTop');
    $window.on('scroll', function () {
        if ($(this).scrollTop() > 900) {
            $(scrolls).fadeIn();
        } else {
            $(scrolls).fadeOut();
        }

    });

    $(scrolls).on('click', function () {
        $('html, body').animate({scrollTop: 0}, 800);
        return false;
    });


//////// -Sticky Header- /////////
    var scroll_position = 0;

    if($(window).width()>768) {
        $window.on('scroll', function () {
            scroll_position = $(this).scrollTop();
            var headers = $('.header');
            if(scroll_position > 150) { // After scrolling above 300px height show the sticky menu
                $(headers).css('position', 'fixed'); // Make header fixed position
                $(headers).css('background', '#1a1919'); // Change the sticky header background
                $(headers).css('z-index', '100');  // Stack order of the header
            } else {
                $(headers).css('position', 'absolute'); // Return to normal header under 300px height scroll
                $(headers).css('background', 'transparent'); // Change to normal transparent background
            }
        });
    } else {
        var headers = $('.header');
        $window.on('scroll', function () {
            scroll_position = $(this).scrollTop();
            if(scroll_position > 150) {
                $(headers).css('position', 'fixed'); // Make header fixed position
            } else {
                $(headers).css('position', 'absolute'); // Return to normal header under 300px height scroll
            }
        });
        $(headers).css('background', '#1a1919'); // Change the sticky header background
        $(headers).css('z-index', '100');  // Stack order of the header
    }

    //////// -Parallax Sections- /////////
    var first_parallax = $('.parallax-window');
    var second_parallax = $('.parallax-adventages');
    var parallax_testimonials = $('.parallax-testimonial');
    var single_image_parallax = $('.single-parallax');
    $(first_parallax).parallax();
    $(second_parallax).parallax();
    $(parallax_testimonials).parallax();
    $(single_image_parallax).parallax();


/*//////// -Slick Slider for Testimonials- /////////
    var testim = $('.testimonials');
    $(testim).slick({
        centerMode: true,
        centerPadding: '60px',
        slidesToShow: 1,  // Default items to show
        dots: true,
        arrows: false,
        autoplay: true,
        autplaySpeed: 2000,
        responsive: [
            {
                breakpoint: 1024,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '60px',
                    slidesToShow: 1
                }
            },
            {
                breakpoint: 768,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '40px',
                    slidesToShow: 1
                }
            },
            {
                breakpoint: 480,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '40px',
                    slidesToShow: 1
                }
            },
            {
                breakpoint: 1024,
                settings: {
                    centerMode: true,
                    slidesToShow: 1,
                    centerPadding: '40px'
                }
            }
        ]
    });

//////// -Slick Slider for Our Team- /////////
    var center = $('.center');
    $(center).slick({
        centerMode: true,
        infinite: true,
        centerPadding: '50px',
        arrows: true,
        autoplay: false,
        autoplaySpeed: 2000,
        slidesToShow: 3, //  Default items to show

        responsive: [{
            breakpoint: 1024,
            settings: {
                arrows: true,
                centerMode: true,
                centerPadding: '60px',
                slidesToShow: 1
            }
        },
            {
                breakpoint: 768,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '40px',
                    slidesToShow: 1
                }
            },
            {
                breakpoint: 480,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '40px',
                    slidesToShow: 1
                }
            },
            {
                breakpoint: 320,
                settings: {
                    arrows: false,
                    centerMode: true,
                    centerPadding: '40px',
                    slidesToShow: 1
                }
            }
        ]
    });*/

})(jQuery);








slider/js/revolution-slider.js

				/******************************************
					-	PREPARE PLACEHOLDER FOR SLIDER	-
				******************************************/			
				
                var tpj = jQuery;

                var revapi8;
                tpj(document).ready(function() {
                    if (tpj("#rev_slider_1_1").revolution == undefined) {
                        revslider_showDoubleJqueryError("#rev_slider_1_1");
                    } else {
                        revapi8 = tpj("#rev_slider_1_1").show().revolution({
                            sliderType: "standard",
                            jsFileLocation:"rs-plugin/js/",
                            sliderLayout: "fullwidth",
                            delay: 9000,
                            navigation: {
                                keyboardNavigation: "on",
                                keyboard_direction: "horizontal",
                                mouseScrollNavigation: "off",
                                onHoverStop: "off",
                                touch: {
                                    touchenabled: "on",
                                    swipe_threshold: 75,
                                    swipe_min_touches: 1,
                                    swipe_direction: "horizontal",
                                    drag_block_vertical: false
                                },
                                arrows: {
                                    style: "uranus",
                                    enable: true,
                                    hide_onmobile: true,
                                    hide_onleave: false,
                                    tmp: '',
                                    left: {
                                        h_align: "left",
                                        v_align: "center",
                                        h_offset: 10,
                                        v_offset: 0
                                    },
                                    right: {
                                        h_align: "right",
                                        v_align: "center",
                                        h_offset: 10,
                                        v_offset: 0
                                    }
                                },
                                bullets: {
                                    enable: true,
                                    hide_onmobile: true,
                                    style: "uranus",
                                    hide_onleave: false,
                                    direction: "horizontal",
                                    h_align: "center",
                                    v_align: "bottom",
                                    h_offset: 20,
                                    v_offset: 30,
                                    space: 5,
                                    tmp: '<span class="tp-bullet-inner"></span>'
                                }
                            },
							visibilityLevels:[1240,1024,778,480],
							gridwidth:1400,
							gridheight:800,
							disableProgressBar: "on",
							shadow:0,
							spinner:"spinner2",
							lazyType:"none",							
							stopLoop:"off",
							stopAfterLoops:-1,
							stopAtSlide:-1,
							shuffle:"off",
							autoHeight:"off",
							disableProgressBar:"on",
							hideThumbsOnMobile:"off",
							hideSliderAtLimit:0,
							hideCaptionAtLimit:0,
							hideAllCaptionAtLilmit:0,
							debugMode:false,
							parallax: {
								type:"mouse",
								origo:"enterpoint",
								speed:400,
								levels:[5,10,15,20,25,30,35,40,45,46,47,48,49,50,51,55],
								type:"mouse",
							},							
                            debugMode: false,
                            fallbacks: {
                                simplifyAll: "off",
                                nextSlideOnWindowFocus: "off",
                                disableFocusListener: false,
                            }
                        });
                    }
                }); /*ready*/				

slider/js/script.js
/*  Table of Contents
01. MENU ACTIVATION
02. FITVIDES RESPONSIVE VIDEOS
03. MOBILE MENU
04. SHOW/HIDE MOBILE MENU
05. GALLERY JS
06. STICKY MENU JS
07. SCROLL TO TOP MENU JS
08. prettyPhoto JS
09. MatchHeight
10. FORM VALIDATION
11. Background Images
12. LIGHT SHORTCODES
13. MASONRY
*/

jQuery(document).ready(function($) {
    'use strict';


/*
=============================================== 01. MENU ACTIVATION  ===============================================
*/
    jQuery('nav#site-navigation ul.sf-menu, .header-top-right-avlar ul.sf-menu').superfish({
        popUpSelector: 'ul.sub-menu,.sf-mega', 	// within menu context
        delay:      	400,                	// one second delay on mouseout
        speed:      	200,               		// faster animation speed
        speedOut:    	200,             		// speed of the closing animation
        animation: 		{opacity: 'show'},		// animation out
        animationOut: 	{opacity: 'hide'},		// adnimation in
        cssArrows:     	true,              		// set to false
        autoArrows:  	true                    // disable generation of arrow mark-up
    });


/*
=============================================== 03. MOBILE MENU  ===============================================
*/
    $('ul.mobile-menu-pro').slimmenu({
        resizeWidth: '1200',
        collapserTitle: 'Menu',
        easingEffect:'easeInOutQuint',
        animSpeed:'medium',
        indentChildren: false,
        childrenIndenter: '- '
    });

/*
=============================================== 04. SHOW/HIDE MOBILE MENU  ===============================================
*/
    var clickOrTouch = (('ontouchend' in window)) ? 'touchend' : 'click';

    $(".mobile-menu-icon-pro").on(clickOrTouch, function(e) {
        $(".site-header-pro").toggleClass("active-menu-icon-pro");
    });



/*
=============================================== 05. GALLERY JS  ===============================================
*/
    $('.gallery-progression').flexslider({
        animation: "fade",
        slideDirection: "horizontal",
        slideshow: false,
        slideshowSpeed: 7000,
        animationDuration: 200,
        directionNav: true,
        controlNav: true,
        prevText: "",
        nextText: "",
    });



/*
=============================================== 06. STICKY MENU JS  ===============================================
*/
    if ($(this).width() > 959) {
        $('#sticky-header-pro').scrollToFixed();
    }

    var header = $(".menu-default-pro");
    $(window).scroll(function() {

        if ($(this).width() > 959) {

            var scroll = $(window).scrollTop();

            if (scroll >= 50) {
                header.removeClass('menu-default-pro').addClass("menu-resized-pro");

            } else {
                header.removeClass("menu-resized-pro").addClass('menu-default-pro');
            }

        }

    });



    $(window).scroll(function(){
        if ($(this).width() > 959) {
            $(".invested_header_footer #invested-bottom-header").css("bottom",Math.max(0,$(this).scrollTop()));
        }
    });



/*
=============================================== 07. SCROLL TO TOP MENU JS  ===============================================
*/
    // browser window scroll (in pixels) after which the "back to top" link is shown
    var offset = 150,

    //browser window scroll (in pixels) after which the "back to top" link opacity is reduced
    offset_opacity = 1200,

    //duration of the top scrolling animation (in ms)
    scroll_top_duration = 700,

    //grab the "back to top" link
    $back_to_top = $('#pro-scroll-top');

    //hide or show the "back to top" link
    $(window).scroll(function(){
        ( $(this).scrollTop() > offset ) ? $back_to_top.addClass('cd-is-visible') : $back_to_top.removeClass('cd-is-visible cd-fade-out');
        if( $(this).scrollTop() > offset_opacity ) {
            $back_to_top.addClass('cd-fade-out');
        }
    });

    //smooth scroll to top
    $back_to_top.on('click', function(event){
        event.preventDefault();
        $('body,html').animate({
            scrollTop: 0 ,
        }, scroll_top_duration
                              );
    });


/*
=============================================== 09. Match Height  ===============================================
*/

    $(function() {
        $('.equal-height-progression').matchHeight({
            byRow: true,
        });
    });


/*
=============================================== 10. Form Validation  ===============================================
*/


    $("#CommentForm").validate();

/*
=============================================== 12. Light Shortcodes  ===============================================
*/


    // Accordion
    $(".ls-sc-accordion").accordion({heightStyle: "content"});

    // Toggle
    $(".ls-sc-toggle-trigger").on('click', function(){$(this).toggleClass("active").next().slideToggle("fast");return false; });

    // Tabs
    $( ".ls-sc-tabs" ).tabs();


/*
=============================================== 13. MASONRY  ===============================================
*/

    // portfolio & team
    var $container_portfolio = $('.progression-portfolio-masonry, .progression-masonry-team').isotope();
    $container_portfolio.imagesLoaded( function() {
        $(".progression-masonry-item").addClass('opacity-progression');
        $container_portfolio.isotope({
            itemSelector: '.progression-masonry-item',
            transitionDuration: '0.4s',

            masonry: {
                columnWidth: '.progression-masonry-col-4',
            },hiddenStyle: {
                opacity: 0
            },
            visibleStyle: {
                opacity: 1
            }
        });
    });




    var $container_press = $('.progression-masonry-press, .progression-single-portfolio-masonry').isotope();
    $container_press.imagesLoaded( function() {
        $(".progression-masonry-item").addClass('opacity-progression');
        $container_press.isotope({
            itemSelector: '.progression-masonry-item',
            transitionDuration: '0.4s',

            masonry: {
                columnWidth: '.progression-masonry-col-3',
            },hiddenStyle: {
                opacity: 0
            },
            visibleStyle: {
                opacity: 1
            }
        });
    });


    $('.port-filter-group-1').on( 'click', 'li', function() {
        var filterValue = $(this).attr('data-filter');
        $container_press.isotope({ filter: filterValue });
    });

    $('.port-filter-group-1').each( function( i, buttonGroup ) {
        var $buttonGroup = $( buttonGroup );
        $buttonGroup.on( 'click', 'li', function() {
            $buttonGroup.find('.is-checked').removeClass('is-checked');
            $( this ).addClass('is-checked');
        });
    });


});
